CHANGELOG.md

```diff
@@ -555,7 +555,7 @@ FEATURES:
 * gaiad init overhaul
   * Create genesis transactions with `gaiad init gen-tx`
   * New genesis account keys are automatically added to the client keybase (introduce `--client-home` flag)
-  * Initialize with genesis txs using `--gen-txs` flag
+  * Initialize with genesis txs using `--with-txs` flag
```

Comment:
Don't think we should modify an old changelog entry

Reply:
++